### ISSUE TITLE
Logging code deploys as metrics

### ISSUE THREAD

**Issue by chrisproud** [GlobeBMG/GBMG.Monitoring#3]
How do we log code deploys as metrics?

**Comment by chrisproud** [comment]
We would like to store code deploy metrics in the same place as app metrics, but surely the metric is stored by octopus rather than the application?

**Comment by msancho** [comment]
Yeah, octopus will generate the event to store (or any other deployment tool we use) but as an application metric, I think, it should be stored in the same way.
